install_config/install/advanced_install.adoc

```diff
@@ -398,6 +398,69 @@ meaning that it is available for placement of new pods. See
 xref:marking-masters-as-unschedulable-nodes[Configuring Schedulability on Masters].
 |===
 
+
+
+[[configuring-cluster-preflight-checks]]
+=== Configuring Cluster Preflight Checks
+
+Preflight checks are a set of diagnostic tasks that run as part of the `OpenShift Health Checks` Ansible role.
```

Comment:
Should we start talking about "pre-install" instead of "preflight"? /cc @sosiouxme